test/cookie.js

```diff
@@ -91,7 +91,8 @@ describe('new Cookie()', function () {
         var maxAge = 60000
         var cookie = new Cookie({ maxAge: maxAge })
 
-        assert.strictEqual(cookie.maxAge, maxAge)
+        assert.ok(cookie.maxAge <= maxAge);
```

Comment:
This check seems wrong. For example, it still passes even when `cookie.maxAge` is `undefined`, which doesn't seem in the spirit of validating that the maxage was actually set.

Reply:
In my test, that statement does fail as expected If `cookie.maxAge` was undefined.
I tested with this code:
```
        cookie.maxAge = undefined;
        assert.ok(cookie.maxAge <= maxAge);
```
and got:
```
      AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:
  assert.ok(cookie.maxAge <= maxAge)
```
Anyway I've added an extra check.